api/errors.go

```diff
@@ -67,6 +67,7 @@ var (
 	ErrNoElectionKeys                   = apirest.APIerror{Code: 4048, HTTPstatus: apirest.HTTPstatusNotFound, Err: fmt.Errorf("no election keys available")}
 	ErrKeyNotFoundInCensus              = apirest.APIerror{Code: 4049, HTTPstatus: apirest.HTTPstatusNotFound, Err: fmt.Errorf("key not found in census")}
 	ErrInvalidStatus                    = apirest.APIerror{Code: 4050, HTTPstatus: apirest.HTTPstatusBadRequest, Err: fmt.Errorf("invalid status")}
+	ErrParticipanKeyLengthInvalid       = apirest.APIerror{Code: 4051, HTTPstatus: apirest.HTTPstatusBadRequest, Err: fmt.Errorf("participant census key length is wrong")}
```

Comment:
whenever possible, keep the var name exactly the same as the string.
suggestions:
ErrKeyLengthInvalid "key length invalid"
or (if Participant is an important word):
ErrParticipantKeyLengthInvalid "participant key length invalid"

Reply:
I have renamed the error to `ErrInvalidCensusKeyLength` and I changed the message to `invalid census key length`, which I think is more descriptive.